gadget/update.go

```diff
@@ -1620,8 +1620,8 @@ func resolveUpdate(oldVol *PartiallyLaidOutVolume, newVol *LaidOutVolume, policy
 			if err != nil {
 				return nil, err
 			}
-			// Nothing to do after filtering
-			if filter != nil && len(resolvedContent) == 0 && len(newStruct.LaidOutContent) == 0 {
+			// No resolved or raw content that would need updating
+			if len(resolvedContent) == 0 && len(newStruct.LaidOutContent) == 0 {
 				continue
```

Comment:
there's no unit test that hits the continue now, maybe there wasn't any even on master? anyway we might want one

Reply:
There are manager tests that go through this, I have changed a test in gadget to run this nonetheless.